packages/kolibri/components/pages/AppBarPage/index.vue

```diff
@@ -146,20 +145,17 @@
         return show;
       },
     },
-    watch: {
-      windowBreakpoint() {
-        //Update the the app bar height at every breakpoint
-        this.appBarHeight = this.$refs.appBar.$el.scrollHeight || 0;
-      },
+    beforeUpdate() {
+      // Update appBarHeight after AppBar is rerendered and updated
+      this.appBarHeight = this.$refs.appBar.$el.scrollHeight || 124;
```

Comment:
With this being by the [`handleWindowResize`](https://github.com/learningequality/kolibri/pull/13007/files#diff-bc65d92949eabd53d17d9ba8b3666a6dca734c335814690d8e673425ae8ea526R189-R192), it probably makes sense to just call it here?